### ISSUE TITLE
Example for Swing Application

### ISSUE THREAD

**Issue by samzurcher** [scenarioo/scenarioo#392]
As a potential user of scenarioo with an existing Java Swing application, I would like to have an example of Scenario in use with a Swing application using a library such as http://docs.codehaus.org/display/FEST/Home . 

h4. Acceptance Criterias
# Example with a freely available Swing Application
# One simple GUI test
# A couple of screenshots of that GUI test in scenarioo

**Comment by bruderol** [comment]
@samzuercher Good idea. Would be great if anybody (you?) likes to contribute such an example?
I or we can help in case there are questions or problems when developing this example.
Curently I do not see that anybody else of the dev team has time to write such an example.

@Fredericson You also thought  about developing another desktop example? What technology you wanted to choose? Was it SWTBot for Eclipse RCP clients? Is there a related story for that?

**Comment by Fredericson** [comment]
@bruderol @samzurcher the issue #387 is about the scenarioo example i would like to provide. The ui technology is eclipse rcp and the SWTBot the ui testing framework.

**Comment by bruderol** [comment]
@Fredericson thank you for clarification.
